minidump-common/src/format.rs

```diff
@@ -272,6 +272,43 @@ pub struct MINIDUMP_MODULE {
     pub reserved1: [u32; 2],
 }
 
+/// A list of unloaded modules in a minidump.
+///
+/// This is the format of the [`MINIDUMP_STREAM_TYPE::UnloadedModuleListStream`]. The individual
+/// [`MINIDUMP_UNLOADED_MODULE`] entries follow this header in the stream.
+///
+/// This struct matches the [Microsoft struct][msdn] of the same name.
+///
+/// [msdn]: https://docs.microsoft.com/en-us/windows/win32/api/minidumpapiset/ns-minidumpapiset-minidump_unloaded_module_list
+#[derive(Debug, Clone, Pread, SizeWith)]
+pub struct MINIDUMP_UNLOADED_MODULE_LIST {
```

Comment:
Can you stick a comment referencing this up in `MINIDUMP_STREAM_TYPE::UnloadedModuleListStream`?